### PR TITLE
Add initial implementation for in-line doc page.

### PR DESCRIPTION
**Pull request by chamindias**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    const bull = <span className={classes.bullet}>•</span>;
+    const url = ``;
+
+    return (
+        <Grid container justify='center'>
+            <Grid item sm={5}>
+                <Card className={classes.card}>
+                    <CardContent>
+                        <Typography variant="h5" component="h2">
+                            Add New Document Inline
```
Comment (mushthaq33) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Lets add formattedMessage for this.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                        </Typography>
+                        <Divider />
+                        <Typography className={classes.title} gutterBottom>
+                            Inline Doc Description
```
Comment (mushthaq33) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Lets add formattedMessage for this.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                </Card>
+
+                <Card className={classes.card}>
+                    <CardContent>
+                        <Typography variant="h5" component="h2">
+                            Add New Document File Or URL
+                        </Typography>
+                        <Divider />
+                        <Typography className={classes.title} gutterBottom>
+                            File Or URL Doc Description
```
Comment (mushthaq33) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Lets add formattedMessage for this.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                                />
+                            </Button>
+                        </Link>
+                    </CardActions>
+                </Card>
+
+                <Card className={classes.card}>
+                    <CardContent>
+                        <Typography variant="h5" component="h2">
+                            Add New Document File Or URL
```
Comment (mushthaq33) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Lets add formattedMessage for this.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx (hunk 1 above)
Comment (chamindias) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Done : https://github.com/wso2/carbon-apimgt/pull/5838/commits/2624afe75f3b9539267673e69cc1b6b78e26c9ab

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx (hunk 2 above)
Comment (chamindias) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Done : https://github.com/wso2/carbon-apimgt/pull/5838/commits/2624afe75f3b9539267673e69cc1b6b78e26c9ab

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx (hunk 3 above)
Comment (chamindias) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Done : https://github.com/wso2/carbon-apimgt/pull/5838/commits/2624afe75f3b9539267673e69cc1b6b78e26c9ab

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx (hunk 4 above)
Comment (chamindias) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Done : https://github.com/wso2/carbon-apimgt/pull/5838/commits/2624afe75f3b9539267673e69cc1b6b78e26c9ab

Comment (mushthaq33) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/DocumentType.jsx]: Great :)

**Pull request merged**